Update windows-autopilot-hybrid.md (#2625)
MicrosoftDocs/memdocs: memdocs/autopilot/windows-autopilot-hybrid.md

4. If you selected **Dynamic Devices** for the membership type, in the **Group** pane, select **Dynamic device members**.

> 5. Click on **Edit** in the **Rule syntax** box, enter one of the following code lines:

Reviewer: ```suggestion
5. Select **Edit** in the **Rule syntax** box and enter one of the following code lines:
```

    - To create a group that includes all your Autopilot devices, enter `(device.devicePhysicalIDs -any _ -contains "[ZTDId]")`.

>     - Intune's Group Tag field maps to the OrderID attribute on Azure AD devices. If you want to create a group that includes all of your Autopilot devices with a specific Group Tag(OrderID), type: `(device.devicePhysicalIds -any _ -eq "[OrderID]:179887111881")`

Reviewer: ```suggestion
    - Intune's Group Tag field maps to the OrderID attribute on Azure AD devices. If you want to create a group that includes all of your Autopilot devices with a specific Group Tag (OrderID), type: `(device.devicePhysicalIds -any _ -eq "[OrderID]:179887111881")`.
```
